federation/cluster/federation-up.sh

```diff
@@ -118,6 +115,7 @@ function init() {
       --apiserver-enable-token-auth=true \
       --apiserver-arg-overrides="--runtime-config=api/all=true,--v=4" \
       --controllermanager-arg-overrides="--v=4" \
+      --etcd-persistent-storage=${USE_PV_FOR_ETCD:-false} \
```

Comment:
Can you please do this defaulting at the top? See https://github.com/kubernetes/kubernetes/blob/master/federation/cluster/common.sh#L25 for example. It's difficult to track it otherwise. Also, can you please prefix the global var passed down from the test infra with `FEDERATION_` to not pollute the namespace.